addons/product_expiry/models/production_lot.py

```diff
@@ -21,6 +21,23 @@ class StockLot(models.Model):
     product_expiry_alert = fields.Boolean(compute='_compute_product_expiry_alert', help="The Expiration Date has been reached.")
     product_expiry_reminded = fields.Boolean(string="Expiry has been reminded")
 
+    @api.depends('use_expiration_date', 'expiration_date', 'alert_date')
+    @api.depends_context('formatted_display_name')
+    def _compute_display_name(self):
+        lots_to_process = []
```

Comment:
To respect [coding guideline](_compute_display_name), it would be cool if these variables are named like that instead
```suggestion
        lots_to_process_ids = []
```
Same for the other computes